config.sh

```diff
@@ -275,7 +275,7 @@ WIN_PERL=${WIN_PERL:-"C:/perl/bin/perl.exe"}
 # Android default settings and tools
 if test "${OS}" = "android" ; then
     ANDROID_NDK_VERSION=${ANDROID_NDK_VERSION:-r10d}
-    ANDROID_PLATFORM=${ANDROID_PLATFORM:-android-10}
+    ANDROID_PLATFORM=${ANDROID_PLATFORM:-android-17}
```

Comment:
Is this going to cause a compatibility problem? Or anything we need to document... Like people installing appropriate API version in their SDKs when building LC standalones?